reporter/src/main/kotlin/model/EvaluatedRuleViolation.kt

```diff
@@ -32,7 +32,7 @@ import org.ossreviewtoolkit.model.config.RuleViolationResolution
 data class EvaluatedRuleViolation(
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     val rule: String,
-    val pkg: EvaluatedPackage,
+    val pkg: EvaluatedPackage?,
```

Comment:
This might require changes in the reporter-web-app.

Reply:
Yes, it does. See also description of the PR.

Reply:
Addressed by #4450 .